invenio_grobid/templates/grobid/index.html

```diff
@@ -71,6 +71,15 @@ <h1 class="pull-left grobid-logo"><a href="{{url_for('grobid.index')}}">GROB<spa
       </dd>
       <dt>Abstract</dt>
       <dd>We present a very simple algorithm for the Least Common Ancestor problem. We thus dispel the frequently held notion that an optimal LCA computation is unwieldy and unimplementable. Interestingly, this algorithm is a sequentialization of a previously known PRAM algorithm of Berkman, Breslauer, Galil, Schieber, and Vishkin.</dd>
+      <dt>Keywords</dt>
+      <dd>
+        <ul>
+          <li>Data Structures</li>
+          <li>Least Common Ancestor (LCA)</li>
+          <li>Range Minimum Query (RMQ)</li>
+          <li>Cartesian Tree</li>
+        </ul>
+      </dd>
```

Comment:
Sorry for this noise in the PR; I was supposed to add this to the keyword extraction PR. Should I remove this and refile as another PR?